packages/core/src/sdk/product/useProduct.ts

```diff
@@ -11,7 +11,7 @@ import { useSession } from '../session'
 
 const query = gql`
   query ClientProductQuery($locator: [IStoreSelectedFacet!]!) {
-    ...pdp
+    ...ClientProductFragment
```

Comment:
Yes, I understood, but we have a problem here, because if the ClientProductQuery turn ClientProduct and ClientProductFragment turn ClientProduct too, we will have a name conflict